include/constants/battle_move_effects.h

```diff
@@ -197,7 +197,7 @@
 #define EFFECT_SNATCH                       193
 #define EFFECT_LOW_KICK                     194
 #define EFFECT_SECRET_POWER                 195
-#define EFFECT_RECOIL_33                    196
+#define EFFECT_UNUSED_196                   196
```

Comment:
Unused status should be removed and everything shifted.

Reply:
Yup, I'm just leaving that until the end so that I don't have to shift multiple times if I remove more